benchmark/poseidon2/poseidon2_benchmark.cc

```diff
@@ -81,37 +82,36 @@ void Run(SimplePoseidonBenchmarkReporter& reporter,
 }
 
 int RealMain(int argc, char** argv) {
-  tachyon::Poseidon2Config config;
+  Poseidon2Config config;
   if (!config.Parse(argc, argv)) {
     return 1;
   }
 
-  SimplePoseidonBenchmarkReporter reporter("Poseidon2 Benchmark",
-                                           config.repeating_num());
-
-  for (const tachyon::Poseidon2Config::Vendor vendor : config.vendors()) {
-    reporter.AddVendor(tachyon::Poseidon2Config::VendorToString(vendor));
-  }
-
-  switch (config.prime_field()) {
-    case tachyon::Poseidon2Config::PrimeField::kBabyBear: {
-      Run<math::BabyBear>(reporter, config, run_poseidon2_horizen_baby_bear,
-                          run_poseidon2_plonky3_baby_bear);
-      break;
-    }
-    case tachyon::Poseidon2Config::PrimeField::kBn254Fr: {
-      Run<math::bn254::Fr>(reporter, config, run_poseidon2_horizen_bn254_fr,
-                           run_poseidon2_plonky3_bn254_fr);
-      break;
-    }
+  SimpleReporter reporter("Poseidon2 Benchmark");
+  reporter.SetXLabel("Trial number");
+  reporter.SetColumnLabels(
+      base::CreateVector(config.repeating_num(),
+                         [](size_t i) { return base::NumberToString(i); }));
+
+  if (config.prime_field().value() == FieldType::kBabyBear) {
+    Run<math::BabyBear>(reporter, config, run_poseidon2_horizen_baby_bear,
+                        run_poseidon2_plonky3_baby_bear);
+  } else if (config.prime_field() == FieldType::Bn254Fr()) {
```

Comment:
Can these if statements be standardized to use only `FieldType::k...` or `FieldType:...()`?